heroku_deploy/entrypoint.sh

```diff
@@ -69,8 +69,19 @@ main(){
     app_name=${INPUT_HEROKU_APP_NAME_STAGING_THREE}
 
   fi
-  
-  git push https://heroku:${INPUT_HEROKU_API_KEY}@git.heroku.com/${app_name}.git HEAD:master -f
+
+  # The preboot will be always active, only when the label “migration” is present will be disabled.
+  # TODO: Once tested, 'staging_3' must be chenged bye 'production'
+  if  [ "${DEPLOY_ENVIRONMENT}" = "staging_3" ] && [ "${WITH_MIGRATION}" = "Y" ]; then
+    heroku features:disable preboot -a ${app_name}
+  fi
+
+  git push https://heroku:${HEROKU_API_KEY}@git.heroku.com/${app_name}.git HEAD:master -f
```

Comment:
INPUT_HEROKU_API_KEY is now HEROKU_API_KEY. is that right?

Reply:
ok, guess it depends on the env/with declaration in the workflow yml

Reply:
@pbaro Yes, your conclusion was correct.